src/test/shared/cloudformation/cloudformation.test.ts

```diff
@@ -136,16 +136,15 @@ describe('CloudFormation', () => {
             assert.deepStrictEqual(loadedTemplate, expectedTemplate)
         })
 
-        it('only loads YAML with valid types', async () => {
-            // timeout is not a number
+        it('only loads YAML with valid fields', async () => {
+            // codeuri is missing
             const badYamlStr: string = `Resources:
     TestResource:
         Type: ${CloudFormation.SERVERLESS_FUNCTION_TYPE}
         Properties:
             Handler: handler
-            CodeUri: codeuri
             Runtime: runtime
-            Timeout: not a number
+            Timeout: 1
```

Comment:
This change implies we don't have a test for each validation check :/ 
Not something that needs to be addressed in this PR, but still, :/